### DISCUSSION TITLE
Package icons not displaying for anyone else?

### DISCUSSION THREAD

**Discussion by secondl1ght** [atom/atom#22614] [Q&A]
Some of the package icons do not load in the app, does anyone else have this and know why?

Thanks

**Comment by DeeDeeG** [comment]
If the package's source repository is a GitHub URL that redirects (such as a repository that has changed owners or moved under a new org account), Atom is apparently not smart enough to follow the redirect and fetch the icon from the new URL. Although when cloning/downloading the package, GitHub.com handles the redirect transparently, so the package will still install.
